ml-agents/mlagents/trainers/tests/test_config_conversion.py

```diff
@@ -0,0 +1,191 @@
+import yaml
```

Comment:
I tried converting the trainer_config.yaml I think it works well. Only feedback is that I would like some spaces between the different behaviors and curriculum / sampler.

Reply:
> A few minor things, but I approve overall.
> Is there a path to deprecation for this file?
> How many releases is it going to stay?

Thanks for the comments, will address before merging.

My thought is 2 releases before deprecation. That should mean the script exists when we publicize the Python CLI (e.g. blog post), but not the release after. Thoughts?

Reply:
> will try to find a workaround.

No need to try too hard. If there is not an easy way to do it, leave it. It does not break anything.

Reply:
Added a comment about this at the top